### ISSUE TITLE
Cityscapes Evaluation

### ISSUE THREAD

**Issue by veroveroxie** [WeilunWang/NEGCUT#1]
Hi, thanks for your work.

Could you please release the drn model and your pre-trained models?

About the cityscapes, do you use the validation set given by CUT or the test set given by running prepare_dataset.py?  Or do you change the training set of this dataset?
(I have tested the pre-trained model of CUT, the results are very weird. When I use the validation set, the FID matches the results in their paper. But if I use the test set, the FID is around 100.)

For the experiments, do you choose the best epoch or the final epoch?

Sorry for so many questions.
Best

**Comment by WeilunWang** [comment]
Hi, thanks for your interest.

1. Regarding the pre-trained model, since I am on holiday
Duplicate of #
Duplicate of #, it's not very convenient for me to get them right now. I will release them later.
2. Same as CUT, I use the validation set given by CUT to evaluate our model. This is because the semantic maps of test set is no t avaliable. (They are all blank in the cityscapes dataset.) This may be why you got a large FID when you test on the test set.
3. On the cityscapes, we choose the final epoch, which is also the best epoch.

**Comment by WeilunWang** [comment]
Hi, thanks for your interest.

Regarding the pre-trained model, since I am on holiday
Duplicate of #, it's not very convenient for me to get them right now. I will release them later.
Same as CUT, I use the validation set given by CUT to evaluate our model. This is because the semantic maps of test set is no t avaliable. (They are all blank in the cityscapes dataset.) This may be why you got a large FID when you test on the test set.
On the cityscapes, we choose the final epoch, which is also the best epoch.

**Comment by WeilunWang** [comment]
Hi, thanks for your interest.

Regarding the pre-trained model, since I am on holiday, it's not very convenient for me to get them right now. I will release them later.
Same as CUT, I use the validation set given by CUT to evaluate our model. This is because the semantic maps of test set is no t avaliable. (They are all blank in the cityscapes dataset.) This may be why you got a large FID when you test on the test set.
On the cityscapes, we choose the final epoch, which is also the best epoch.

**Comment by WeilunWang** [comment]
Hi, thanks for your interest.

1. Regarding the pre-trained model, since I am on holiday, it's not very convenient for me to get them right now. I will release them later.
2. Same as CUT, I use the validation set given by CUT to evaluate our model. This is because the semantic maps of test set is no t avaliable. (They are all blank in the cityscapes dataset.) This may be why you got a large FID when you test on the test set.
3. On the cityscapes, we choose the final epoch, which is also the best epoch.

**Comment by veroveroxie** [comment]
Thanks for your quick reply!

For the cityscapes dataset, CUT used this resized and JPEG-version. Do you use this tar file in your evaluation? 

"""
Note: the Cityscapes pretrained model was trained and evaluated on a resized and JPEG-compressed version of the original Cityscapes dataset. To perform evaluation, please download [this](http://efrosgans.eecs.berkeley.edu/CUT/datasets/cityscapes_val_for_CUT.tar) validation set and perform evaluation.
"""

**Comment by WeilunWang** [comment]
Hi,

You can send a email to me, I will reply you with my test demo.

Best,
Weilun

**Comment by Lucky0775** [comment]
> Thanks for your quick reply!
> 
> For the cityscapes dataset, CUT used this resized and JPEG-version. Do you use this tar file in your evaluation?
> 
> """ Note: the Cityscapes pretrained model was trained and evaluated on a resized and JPEG-compressed version of the original Cityscapes dataset. To perform evaluation, please download [this](http://efrosgans.eecs.berkeley.edu/CUT/datasets/cityscapes_val_for_CUT.tar) validation set and perform evaluation. """

Hi,have you solved this problem ? I also have the same confusion.
